src/pages/Account/AccountVerification.tsx

```diff
@@ -1,11 +1,173 @@
+            </ListGroup>
+          );
+        }
+
+        return (
+          <>
+            <Alert
+              isOpen={alert.show}
+              color={alert.color}>
+              {alert.msg}
+            </Alert>
+
+            <ListGroup className="d-flex flex-column">
+              <ListGroupItem
+                className="mb-5"
+                onClick={toggleModal}>
+                <CardBody>
+                  <h6 className="mb-0">Phone (Required){' '}
+                    {phoneVerified
+                      ? <span className="small text-success">(Verified)</span>
+                      : <span className="small text-danger">(Not Verified)</span>}
+                  </h6>
+                  <div className="d-flex justify-content-between">
+                    <h6 className="mb-0 small text-muted">
+                      {phoneVerified
+                        ? VERIFICATION_MESSAGE.PHONE.CHANGE
+                        : VERIFICATION_MESSAGE.PHONE.ADD }
+                    </h6>
+                    <i className="fa fa-lock"></i>
+                  </div>
+                </CardBody>
+              </ListGroupItem>
+
+              <ListGroupItem
+                disabled={emailVerified}
+                onClick={handleEmailVerification}>
+                <CardBody>
+                  <h6 className="mb-0">Email (Required){' '}
+                    {emailVerified
+                      ? <span className="small text-success">(Verified)</span>
+                      : <span className="small text-danger">(Not Verified)</span>
+                    }
+                  </h6>
+                  <div className="d-flex justify-content-between">
+                    <h6 className="mb-0 small text-muted">
+                      {emailVerified
+                        ? VERIFICATION_MESSAGE.EMAIL.VERIFIED
+                        : VERIFICATION_MESSAGE.EMAIL.VERIFY }
+                    </h6>
+                    <i className="fa fa-lock"></i>
+                  </div>
+                </CardBody>
+              </ListGroupItem>
+            </ListGroup>
+
+            <PhoneNumberVerificationForm
+              isOpen={isOpen}
+              toggleModal={toggleModal}
+              refreshVerificationStatus={refreshVerificationStatus}
+              user={user} />
+          </>
+        )
+      }}
+    </FirebaseConsumer>
+  );
+
+  function handleEmailVerification() {
+    if (!auth.currentUser) {
+      return setAlert({
+        color: 'danger',
+        msg: 'Could not find current user.',
+        show: true,
+      });
+    }
+
+    auth.currentUser.sendEmailVerification()
+      .then(() => {
+        setAlert({
+          color: 'success',
+          msg: getDisplaySuccessMessage(SuccessMessage.EMAIL_VERIFICATION_SENT),
+          show: true,
+        })
+      })
+      .catch((error: any) => {
+        console.error(error);
+        setAlert({
+          color: 'success',
+          msg: error,
```

Comment:
Should this be `error.message`? Not sure what the failure cases are for `sendEmailVerification`

Reply:
ah thanks!